src/index.js

```diff
@@ -67,6 +72,10 @@ render(
 
     <DonutWidget widget={donutWidget} />
 
+    <br />
+
+    <StackedColumnWidget widget={stackedColumnWidget} />
```

Comment:
Lets add a prop here for `isPercentage` (or better name) to allow us to configure either full height stacking columns or not full height. Then we can consume that configuration in the chart initialisation like we so with `widget` to flick between stack percentage and normal.
 
![screen shot 2017-04-05 at 11 41 02 am](https://cloud.githubusercontent.com/assets/2195815/24686072/cf125414-19f4-11e7-98fe-8eeed947d162.png)
![screen shot 2017-04-05 at 11 40 59 am](https://cloud.githubusercontent.com/assets/2195815/24686073/cf13ff1c-19f4-11e7-84e5-4d7d4b2fd229.png)